docs/UserGuide.md

```diff
@@ -32,103 +33,138 @@
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
 > * The order of parameters is fixed.
 
 #### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding an item: `add`
+Adds an item into the task manager<br>
+
+*With deadline*
+
+Format: `add NAME by DUE_DATE DUE_TIME [repeat every RECURRING_INTERVAL]`
+>Notice the `by` keyword? We use `by` to denote a deadline.
+
+Examples:
+* `add Do project proposal by 5pm tomorrow`
+* `add Buy coffee for boss by 7am repeat every day`
+
+*Without deadline*
+
+Format: `add NAME [rank PRIORITY]`
+>No deadlines? No worries. DearJim allows you to rank your priorities. Assign rank of 1 to 10, with 1 being the most important task. If you did not specify the priority, it will be assigned to a default value of 5.
+
+Example:
+* `add Buy coffee powder rank 1`
+
+*With time interval*
+
+`add NAME at DATE START_TIME to [DATE] END_TIME [repeat every RECURRING_INTERVAL]` 
+> For events, meetings, we use `at`, `to` to specify the date/time interval.
+
+Example: 
+* `add Company meeting tonight at 7pm to 9pm`
```

Comment:
In this case the task name is "Company meeting tonight", date is not specified which defaults to current day. Reading this can be confusing for user though